tests/main/lxd-mount-units/task.yaml

```diff
@@ -30,6 +30,11 @@ execute: |
         lxd.lxc config set core.proxy_https "$http_proxy"
     fi
 
+    # There isn't an image for kinetic yet, let's just check that lxd snap can be installed
+    if os.query is-ubuntu 22.10; then
```

Comment:
Images are available from the community image server from lxd: `lxc launch images:ubuntu/kineti` will work

Reply:
@mvo5, nice, thanks for the suggestion, I updated to use all those image, the only problem is that the images are not coming with snapd installed, so I had to update the tests to support that.